scripts/version_bump.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+	"strconv"
+)
+
+const versionFile = "../internal/version/version.go"
```

Comment:
nit: can this be an arg? It could make unit testing easier

Reply:
Done! Added an extra parameter ```-f``` which takes the path to file